core/controllers/contributor_dashboard.py

```diff
@@ -700,3 +703,190 @@ def post(self):
+                stats = suggestion_services.get_all_question_review_stats(
+                    user_id)
+                self.values = {
+                    'question_review_stats': _get_complete_stats(stats, False)
+                }
+
+        self.render_json(self.values)
+
+
+class ContributorAllStatsSummariesHandler(base.BaseHandler):
+    """Provides all data about submitter and reviewer statistics."""
+
+    GET_HANDLER_ERROR_RETURN_TYPE = feconf.HANDLER_TYPE_JSON
+    URL_PATH_ARGS_SCHEMAS = {
+        'username': {
+            'schema': {
+                'type': 'basestring'
+            }
+        }
+    }
+    HANDLER_ARGS_SCHEMAS = {
+        'GET': {}
+    }
+
+    @acl_decorators.can_fetch_all_contributor_dashboard_stats
+    def get(self, username):
+        """Handles GET requests."""
+
+        user_id = user_services.get_user_id_from_username(username)
+
+        stats = suggestion_services.get_all_contributor_stats(user_id)
+        response = {}
+
+        if stats.translation_contribution_stats is not None:
+            response['translation_contribution_stats'] = _get_complete_stats(
+                stats.translation_contribution_stats, True)
+
+        if stats.translation_review_stats is not None:
+            response['translation_review_stats'] = _get_complete_stats(
+                stats.translation_review_stats, False)
+
+        if stats.question_contribution_stats is not None:
+            response['question_contribution_stats'] = _get_complete_stats(
+                stats.question_contribution_stats, False)
+
+        if stats.question_review_stats is not None:
+            response['question_review_stats'] = _get_complete_stats(
+                stats.question_review_stats, False)
+
+        self.render_json(response)
+
+
+def _get_complete_stats(
+    stats_data: List[Union[
+        suggestion_registry.TranslationContributionStats,
+        suggestion_registry.TranslationReviewStats,
+        suggestion_registry.QuestionContributionStats,
+        suggestion_registry.QuestionReviewStats
+    ]],
+    stats_are_translation_contribution
+):
+    """Returns corresponding stats dicts with all the necessary
+    information for the frontend.
+
+    Args:
+        stats_data: list. Stats domain objects.
+        stats_are_translation_contribution: bool. A flag that indicates whether
+            stats are translation contributions.
+
+    Returns:
+        list. Dict representations of TranslationContributionStats/
+        TranslationReviewStats/QuestionContributionStats/
+        QuestionReviewStats domain objects with additional keys:
+            topic_name: str. Topic name.
+            contribution_months: str. Unique translation contribution
+                months of format: "%b %Y", e.g. "Jan 2021".
+        Unnecessary keys topic_id, contribution_dates, contributor_user_id
+        are consequently deleted.
+    """
+    translation_review_stats_dicts = [
```

Comment:
```suggestion
    stats_dicts = [
```
Since it's not just translation review stats.

Reply:
Done